beautifultable/beautifultable.py

```diff
@@ -1213,3 +1213,53 @@ def from_csv(self, file_name, header=True, **kwargs):
             for row in csv_reader:
                 self.rows.append(row)
             return self
+
+    def to_df(self):
+        """Export table to dataframe.
+
+        Parameters
+        ----------
+        table : BeautifulTable
+
+        """
+        try:
+            pd = __import__("pandas")
+        except ModuleNotFoundError as e:
+            raise ModuleNotFoundError("Please Install pandas to use this API") from e
+
+        # If there are column headers then it will act as a column of datafarme
+        headers = list(self.columns.header)
+        if headers.count(None) == len(headers):
+            headers = None
+
+        # If there are row headers then it will act as an Index
+        index = list(self.rows.header)
+        if index.count(None) == len(index):
+            index = None
```

Comment:
Have we considered the case where some values in the headers are `None` and some aren't? what happens then? pandas wouldn't allow duplicate indexes so would it raise an exception?